test/test.go

```diff
@@ -28,6 +28,80 @@ func testEmbed(g abstract.Group, rand cipher.Stream, points *[]abstract.Point,
 	*points = append(*points, p)
 }
 
+func testPointSet(g abstract.Group, rand cipher.Stream) {
+	N := 1000
+	null := g.Point().Null()
+	for i := 0; i < N; i++ {
```

Comment:
Just to be sure (that we didn't just pick a point for which the test passes). Adding randomized tests is in fact good practice. If you the implementations (sth the test shouldn't really care/should be agnostic about) it's impossible that for one point/scalar the test passes but not for other. 
But if you have strong reason to remove it (besides adding a rather negligible amount of extra time, especially compared to the overall test time of the crypto package), I'm happy to remove it.